pkg/jobrunaggregator/jobrunaggregatorlib/gcs_client.go

```diff
@@ -115,6 +118,10 @@ func (o *ciGCSClient) ReadJobRunFromGCS(ctx context.Context, jobGCSRootLocation,
 	if err := query.SetAttrSelection([]string{"Name", "Created", "Generation"}); err != nil {
 		return nil, err
 	}
+
+	// NOTE: StartOffset and EndOffset are strings that the GCS api uses to limit where you
```

Comment:
nit, isn't the explanation of these simply an IDE hover for their godoc?  Those who don't use modern IDEs are punished accordingly ;)

Reply:
Good point; if I over on StartOffset, it takes me to pkg.go.dev and I see what you mean.